PostHog/posthog
# Fix cohort creation from trends

## Changes

1. In the frontend, instead of creating a new cohort we were patching a cohort with ID `personsModalNew`. Fixed this.
2. The query for adding people had been broken since https://github.com/PostHog/posthog/commit/a8f46b056b44b1cd15f60558f76231f1ec24143a#diff-8c1a8fdf464c1e77c60c9d3a113fa05e04ba342aa680324f9d037e936f3dfd6cR245

## How did you test this code?

Manual QA
1. go to trends, click on a data point (Clickhouse instance)
2. Click 'save as cohort'
3. Add name
4. Click on the link in toast
5. verify the users are in the cohort

Also un-commented the backend test.
